api/namex/services/name_request/exceptions.py

```diff
@@ -128,3 +128,8 @@ def __init__(self, wrapped_err=None, message="Error getting user id."):
 class VirtualWordConditionServiceError(NameRequestException):
     def __init__(self, wrapped_err=None, message="Error initializing VirtualWordCondition Service."):
         super().__init__(wrapped_err, message)
+
+###################rajan exception raising for existing by same name and by same user
+class NameRequestIsAlreadySubmittedError(NameRequestException):
+    def __init__(self, wrapped_err=None, message="The request with same name is already in process."):
```

Comment:
I think "in process" here means starting examining. It might be more precise to use 'submitted' instead..